builtin/logical/transit/path_keys.go

```diff
@@ -154,6 +154,10 @@ func (b *backend) pathPolicyRead(
 			"deletion_allowed":       p.DeletionAllowed,
 			"min_decryption_version": p.MinDecryptionVersion,
 			"latest_version":         p.LatestVersion,
+			"supports_encryption":    p.Type.EncryptionSupported(),
```

Comment:
I'm actually not sure that switching it makes it consistent. :-) 

But, I just checked, and the Vault CLI sorts keys. So potentially keeping it as a block might be nice for scanning the output.